src/pages/static/faq.tsx

```diff
@@ -1,9 +1,107 @@
-const FAQ = () => {
+import { LandingFooter, LandingNav } from '@/components/components';
+import faqData from '@/data/faq.json';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+} from '@chakra-ui/accordion';
+import Image from 'next/image';
+
+export default function FAQ () {
   return (
-    <div>
-      This is the FAQ page
+    <main className="w-screen overflow-hidden lg:block h-fit gap-y-10 dark:bg-neutral-950">
+      <div className="fixed z-40 w-full transition">
+        <LandingNav />
+      </div>
+      <main className="px-6 py-12 md:px-12 max-w-7xl mx-auto grid gap-20">
+        <section className="mt-20 text-center grid gap-3 place-items-center text-zinc-900">
+          <h1 className="text-4xl sm:text-5xl md:text-6xl font-extrabold max-w-4xl">
+            Gyanaguru’s Frequently Asked Question
+          </h1>
+          <p className="text-lg font-medium">
+            Have a question in mind? We've got the answers!
+          </p>
+        </section>
+        <FaqSection />
+        <StillHaveQuestions />
+      </main>
+      <LandingFooter />
+    </main>
+  );
+};
+
+
+function FaqSection () {
+  return (
+    <div className="space-y-20">
+      <form className="grid gap-4 max-w-xs mx-auto border-b border-zinc-300">
+        <div className="flex items-center gap-2 rounded p-1">
+          <span className="material-icons">search</span>
```

Comment:
in all material icons, replace `material-icons` class with `material-symbols-outlined`